### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by brinxmat**

### CODE REVIEW

File: ontology.ttl [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+# Roles
+
+:CreatorRole a rdfs:Class ;
+  rdfs:comment "The base class for all creator roles" ;
+  rdfs:subClassOf :Role .
+
+:ContributorRole a rdfs:Class ;
+  rdfs:comment "The base class for all contributor roles" ;
+  rdfs:subClassOf :Role .
+
```
Comment (gregorgabriel) [inline ontology.ttl:87]: Which fields are mandatory? How and where to define that?

File: ontology.ttl [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# Relations
+# Basic relations
+
+:hasIdentifier a rdf:Property ;
```
Comment (gregorgabriel) [inline ontology.ttl:91]: What is the scope of an identifier? (We have to keep track of the identifying system the identifier refers to)

**Review by gregorgabriel — COMMENTED** [review]
Thats a good start to define the datamodell. We still need a translation into a simple json or/and a definition for the dynamo table database.
Looking forward to see it in action :)

File: Examples/full_example.json [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  "@context": {
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@graph": [
+    {
+      "@id": "_:N8a1bc54d39844a5ab0d89dc0d7c09c94",
+      "@type": "http://nva.unit.no/ontology#FileSet",
+      "http://nva.unit.no/ontology#hasFile": {
```
Comment (janbj) [inline Examples/full_example.json:11]: Should FileSet have more metadata?

File: Examples/full_example.json [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+          "@language": "nb",
+          "@value": "En reise mot en valid datamodell"
+        }
+      ],
+      "http://nva.unit.no/ontology#hasType": {
+        "@id": "http://nva.unit.no/ontology#JournalArticle"
+      }
+    },
+    {
+      "@id": "_:N05c67ed810634949b628581971edc9f3",
```
Comment (janbj) [inline Examples/full_example.json:65]: No type? Anonymous node?

**Pull request closed without merging**